### PR TITLE
Create GUIDE.md

### PR DESCRIPTION
**Pull request by scottux**
Adds a User Manual.

From #502

### CODE REVIEW

**Comment by scottux** [comment]
TODO: Parse to HTML as part of deployment and serve at https://thunderdome.dev/guide

File: docs/GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
+# A Warrior's Guide to Thunderdome
+
+Thunderdome is a fun way to perform agile scrum planning. See Wikipedia.
```
Comment (StevenWeathers) [inline docs/GUIDE.md:3]: Thunderdome is agile scrum practices including Story pointing (battles), Sprint Retrospectives, Story mapping, Async Daily Standup (team standup).

File: docs/GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+As a new warrior in this realm, let this be your guide to Battles. First, we need to know who you are.
```
Comment (StevenWeathers) [inline docs/GUIDE.md:5]: We're more than just pointing battles ;)

File: docs/GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+Use the email/password you created when registering.
+
+- Email
+- Password
+
+_OIDC Providers coming soon._
+
+### Password Retrieval
+
+Forgot your password? Thunderdome can send a link to your email.
```
Comment (StevenWeathers) [inline docs/GUIDE.md:32]: `can send a password reset link to your email` not to be confused with login by email which is a feature we may implement in the future.

File: docs/GUIDE.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+This will be visible to others.
+- Email
+Update your account email, or enter one if you are a guest.
+- Country (optional)
+- Locale, default: English
+8 locales to choose from.
+- Company (optional)
+- Job Title (optional)
+- Theme, default: auto
+If you prefer a darker or lighter interface, you may choose it here.
```
Comment (StevenWeathers) [inline docs/GUIDE.md:50]: Auto lets the operating system and browser define dark or light, if supported.

File: docs/GUIDE.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+- Option: Enable Battle Notification, default: true
```
Comment (StevenWeathers) [inline docs/GUIDE.md:51]: Whether or not notifications show up for events during the battle

File: docs/GUIDE.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+
+### API Access
+
+Create an API key to integrate Thunderdome with your tools. 
+
+See API Documentation here [Thunderdome API Docs](https://thunderdome.dev/swagger/index.html)
+
+### Jira Integration (premium only)
+
+Integrate directly with your team's backlog.
```
Comment (StevenWeathers) [inline docs/GUIDE.md:63]: `to import your stories to point`

File: docs/GUIDE.md [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+
+_Other integrations coming soon._ 
+
+### Delete Account
+
+This is the Thunderdome, but you are free to leave. We will erase all data about you.
```
Comment (StevenWeathers) [inline docs/GUIDE.md:69]: `This is permanent: All poker sessions, retros, story maps, and orgs/teams directly owned by your account will also be deleted.`

File: docs/GUIDE.md [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+
+See who is voting or become a spectator.
+
+### Invite
+
+Send a link for others to join the battle.
+
+## Retrospectives
+
+Review your battles.
```
Comment (StevenWeathers) [inline docs/GUIDE.md:141]: It's not a review of battle, but an agile sprint retrospective.

File: docs/GUIDE.md [hunk 9]
```diff
@@ -0,0 +1,229 @@
...
+
+- Name
+- Link
+- Points
+- Color
+- Content
+- Discussion
+
+### Personas
+
```
Comment (StevenWeathers) [inline docs/GUIDE.md:197]: `*Coming soon`

**Comment by StevenWeathers** [comment]
Provided some feedback, also you missed an entire feature (Teams and Team Standups (asynchronous daily standup tool to aid in speeding up standup or making standups completely async depending on team practices).

**Comment by scottux** [comment]
> you missed an entire feature

Work. In. Progress. :)

No, this is good feedback that I needed to get the details right. I will include those later today.

File: docs/GUIDE.md (hunk 5 above)
Comment (scottux) [reply docs/GUIDE.md:51]: Could it be marketed as "distraction-free mode?"

File: docs/GUIDE.md (hunk 2 above)
Comment (scottux) [reply docs/GUIDE.md:5]:
```suggestion
As a new warrior in this realm, let this be your guide. First, we need to know who you are.
```

File: docs/GUIDE.md (hunk 3 above)
Comment (scottux) [reply docs/GUIDE.md:32]:
```suggestion
Forgot your password? Thunderdome can send a password reset link to your email.
```

File: docs/GUIDE.md (hunk 4 above)
Comment (scottux) [reply docs/GUIDE.md:50]:
```suggestion
The default lets the operating system and browser define dark or light, if supported. If you prefer a darker or lighter interface, you may choose it here.
```

File: docs/GUIDE.md (hunk 6 above)
Comment (scottux) [reply docs/GUIDE.md:63]:
```suggestion
Integrate directly with your team's backlog to import your stories to point.
```

File: docs/GUIDE.md (hunk 7 above)
Comment (scottux) [reply docs/GUIDE.md:69]:
```suggestion
This is the Thunderdome, but you are free to leave. We will erase all data about you.

**This is permanent:** All poker sessions, retros, story maps, and orgs/teams directly owned by your account will also be deleted.
```

File: docs/GUIDE.md (hunk 8 above)
Comment (scottux) [reply docs/GUIDE.md:141]:
```suggestion
Facilitates an agile sprint retrospective.
```

File: docs/GUIDE.md (hunk 9 above)
Comment (scottux) [reply docs/GUIDE.md:197]:
```suggestion

_Coming Soon_

```

File: docs/GUIDE.md (hunk 5 above)
Comment (scottux) [reply docs/GUIDE.md:51]:
```suggestion
- Option: Enable Battle Notification, default: true
Whether or not notifications show up for events during the battle. Disable for distraction-free mode.
```

**Comment by scottux** [comment]
@StevenWeathers This is ready for another round of review.

**Review by StevenWeathers — APPROVED** [review]

**Pull request merged**